blogpost.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Big Jim's Big Ol' Blog Post</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="css/remock.css">
+    <!--Text Font-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <!--Header Font-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@900&display=swap" rel="stylesheet"> 
+    <!--Icon Font-->
+    <script src="https://kit.fontawesome.com/5e40fbdb20.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+       <!---Header-->
+      <section id="header">
+            <a href="/index.html"><h1>BIG OL' BLOG POST</h1></a>
+            <h5>YA YEET</h5>
+            <hr>
+            <hr>
+      </section>
+
+      <!--Iframe and Blog Page-->
+      <section id="main-page">
+        <!--Iframe-->
+        <div class="blog-page-top">
+          <h3><span class="bold">GOTCHA</span> KID</h3>
+          <iframe src="https://player.vimeo.com/video/148751763?autoplay=1&loop=1&autopause=0" width="800" height="480" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen></iframe>
+          <p><a href="https://vimeo.com/148751763">Rick Astley - Never Gonna Give You Up</a> from <a href="https://vimeo.com/user46726126">Rick Astley</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
+        </div>
```

Comment:
Ah, the legend himself. One issue however that this video is causing in it's current layout/styling, it that the whole page does not line up cleanly both horizontally and vertically, giving that nice center focused look of the remock screenshot version of this page. I would either make to media screen wider, or bring the rest of the page (text, horizontal lines, sub headers etc..).